willyserge/broadcaster
# #169828861 build get red-flags route

#### What does this PR do?
have a working endpoint for  getting red-flag records created by a user
#### Description of Task to be completed?
working endpoint 
- /api/v1/red-flags
#### How should this be manually tested?
    - run npm install to install all dependencies
   - run npm start and server
  -  see your port number the server is running on
  - go to postman and make a request to http://localhost: port number /api/v1/red-flags
